### PR TITLE
GraphQL: Add support for variable definitions

### PR DESCRIPTION
**Pull request by stubailo**

### CODE REVIEW

File: tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+query short($foo: ComplexType, $site: Site = MOBILE) {
+  hello
+}
+
+query long(
+  $foo: ComplexType,
+  $site: Float = 124241.12312,
+  $bar: String = "Long string here",
```
Comment (vjeux) [inline tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap]: Does it make sense to respect singleQuote option here?

**Comment by vjeux** [comment]
You'll have to rebase i'm afraid

File: tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (stubailo) [inline tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap]: GraphQL doesn't support single quotes unfortunately :(

**Comment by stubailo** [comment]
Rebased! Will be more careful next time, thanks for being so helpful.

**Comment by vjeux** [comment]
Oh no, it needed to be rebased because I merged another PR which also touched similar code. Keep sending those, they are awesome!

File: tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (vjeux) [inline tests/graphql_variable_definitions/__snapshots__/jsfmt.spec.js.snap]: Well, that's great, one less thing to worry about :p

**Pull request merged**